sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job.py

```diff
@@ -306,9 +306,13 @@ def _from_rest_object(  # pylint: disable=too-many-return-statements
                     return ImportJob._load_from_rest(obj)
 
                 res_command: Job = Command._load_from_rest_job(obj)
+                if hasattr(obj, "name"):
```

Comment:
Nit: should we add a TODO for long term solution? Currently user will still see a warning about the lower node name thing.

Reply:
There is an open item in our backlog to fix it.
https://msdata.visualstudio.com/Vienna/_workitems/edit/3213922